CompassSDK/Tracker/Core/CompassTracker.swift

```diff
@@ -408,6 +408,7 @@ private extension CompassTracker {
         operationQueue.operations.forEach{ $0.cancel() }
         trackInfo.pageUrl = pageName
         pageVars.removeAll()
+        trackInfo.sessionId = storage.sessionId
```

Comment:
https://github.com/Marfeel/MarfeelSDK-iOS/blob/main/CompassSDK/Storage/CompassStorage.swift#L96

we check for expiration every time we read sessionId from model, redefining it on restart, called insided trackNewPage.